### DISCUSSION TITLE
How to get PII permissions

### DISCUSSION THREAD

**Discussion by WWangYu** [amzn/selling-partner-api-models#3059] [General] [answered]
For the last six months, we have been trying to apply for PII permissions, but have been denied, the response: "we only consider sharing Customer data ("Restricted access") with third party Developers that offer features that are materially different from existing applications" , we have done our best to comply with Amazon's policy, but have not received any effective response, any other good suggestions? Or can anyone tell me how FBM's orders are fulfilled without PII permissions. Is Amazon closing the door on PII?

**Comment by WWangYu** [comment]
All the account managers and helpers we could contact seem to refuse to answer this question and get only vague answers, and it seems that a resolution and consensus has been reached within Amazon not to open PII access to any program anymore . If you want permission, you can only buy it from the developer who has the permission. We really can't understand how to fulfill an order without the user's address.

**Comment by LindaNC** [comment]
Hello @WWangYu,

Thank you for bringing up the issue regarding the delay experienced with your {registration/appeal/PII Access request} process. So we can contact the appropriate team, please write back to us with the Registration Case ID.

Thanks,
Linda
Selling Partner API Developer Support

**Comment by WWangYu** [comment]
> Hello @WWangYu,
> 
> Thank you for bringing up the issue regarding the delay experienced with your {registration/appeal/PII Access request} process. So we can contact the appropriate team, please write back to us with the Registration Case ID.
> 
> Thanks, Linda Selling Partner API Developer Support

Thank you for your reply, I have reapplied, the id is: 10302865691

**Comment by WWangYu** [comment]
@LindaNC  

I got rejected again, can you give some advice on how we can get through.

**Comment by WWangYu** [comment]
Is there anyone still paying attention to this issue?

**Reply by NA-Dev** [reply]
No, no they are not.

**Comment by jaimezac27** [comment]
I also have the same problem, I get standard answer and don't know what to do to get the PII.

The case ID is 10564003241. 

Can anyone help me?

**Comment by maxhohenlohe** [comment]
Seems to be an ongoing problem. I've been denied 3 times. Having access to PII would make printing shipping labels a bit easier for us. I may end up creating a bot running selenium and just scrape the orders that way.

**Comment by tianfengLiang** [comment, marked as answer]
We encountered the same issue. Is there any solution?

**Reply by leonardonakagawa** [reply]
Hi @tianfengLiang,

Please be aware that to get PII roles you need to be following the [Amazon Acceptable Use Policy](https://sellercentral.amazon.com.br/mws/static/policy?documentType=AUP&locale=en_US) and [Amazon Data Protection policies](https://sellercentral.amazon.com.br/mws/static/policy?documentType=DPP&locale=en_US).

You can also find details regarding this in our [Developer Profile Review documentation](https://developer-docs.amazon.com/sp-api/docs/register-as-a-public-developer#developer-profile-review).
If the information available in the documentation link provided above does not fully resolve your inquiry, please [open a support case](https://developer.amazonservices.com/support) with us.

Thanks,
Leonardo Nakagawa
Solutions Architect, Selling Partner API

**Reply by tianfengLiang** [reply]
Hi leonardonakagawa,

Thank you for your response and the helpful information. We have carefully reviewed the Amazon Acceptable Use Policy and Amazon Data Protection policies, and we have also reached out multiple times through support cases. However, we are still unable to determine the exact reason for the rejection. We have been applying for a whole year, and our team has spent a significant amount of time on this, yet we are still unclear on what specifically needs to be adjusted.

If there is a particular issue that does not meet the requirements, could you kindly provide us with the detailed reason so that we can make the necessary changes? This is the case ID we recently contacted you with: 10615791282.

We sincerely appreciate your guidance and any further assistance to help us move forward.

Thank you for your time and support.

Best regards,
